x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/all/actions.tsx

```diff
@@ -65,6 +69,9 @@ export const deleteRulesAction = async (
         errors.map(e => e.error.message),
         dispatchToaster
       );
+    } else {
+      // FP: See https://github.com/typescript-eslint/typescript-eslint/issues/1138#issuecomment-566929566
+      onRuleDeleted?.(); // eslint-disable-line no-unused-expressions
```

Comment:
Thanks for the link when putting the disable in 👍 , really helpful and interesting to read.

Reply:
Yeah, they're stilling working out some quirks with linting these new language features it seems.  FWIW it was happy with the below, but I wanted the code feng shui.... 😬

``` ts
if (onRuleDeleted) {
  onRuleDeleted();
}